### PR TITLE
Change AMA to point to dev branch and include configuration of autosens

### PR DESCRIPTION
**Pull request by eyim**

### CODE REVIEW

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode-and-Advanced-Meal-Assist.md [hunk 1]
```diff
@@ -311,7 +311,25 @@ Bottom line == based on insulin only
 
 If no carbs are onboard, then you will have only ONE line.
 
+Currently, there is an upper and lower bound on how much autosens can adjust things.  The defaults are from 0.7x to 1.5x your pump settings.  If you would like to change that, it can be configred in the /lib/profile/index.js file found wherever you have oref0 installed on your device.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode-and-Advanced-Meal-Assist.md]: No, that is the code that sets the defaults.  You want to instead override it in your preferences.json in your openaps config directory (what used to be called max_iob.json, and still should contain that).

**Comment by eyim** [comment]
Ahh - okay.  I guess I am behind a few versions.  I still have the max_iob.json file and that is all it contains.

**Comment by scottleibrand** [comment]
Here is the default preferences.json:
```
{
        "max_iob": 0,
        "type": "current",
        "max_daily_safety_multiplier": 3,
        "current_basal_safety_multiplier": 4,
        "autosens_max": 1.5,
        "autosens_min": 0.7,
        "autosens_adjust_targets": true,
        "override_high_target_with_low": false
}
```

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode-and-Advanced-Meal-Assist.md [hunk 2]
```diff
@@ -311,26 +311,23 @@ Bottom line == based on insulin only
 
 If no carbs are onboard, then you will have only ONE line.
 
-Currently, there is an upper and lower bound on how much autosens can adjust things.  The defaults are from 0.7x to 1.5x your pump settings.  If you would like to change that, it can be configred in the /lib/profile/index.js file found wherever you have oref0 installed on your device.
+Currently, there is an upper and lower bound on how much autosens can adjust things.  The defaults are from 0.7x to 1.5x your pump settings.  If you would like to change that, it can be configred in your preferences.json file.
 
-Looks for this code and adjust the autosens_max and autosens_min as needed
+You preferences.json file will look something like this:
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode-and-Advanced-Meal-Assist.md]: *your

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode-and-Advanced-Meal-Assist.md [hunk 3]
```diff
@@ -311,8 +311,23 @@ Bottom line == based on insulin only
 
 If no carbs are onboard, then you will have only ONE line.
 
+Currently, there is an upper and lower bound on how much autosens can adjust things.  The defaults are from 0.7x to 1.5x your pump settings.  If you would like to change that, it can be configred in your preferences.json file.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode-and-Advanced-Meal-Assist.md]: *configured

**Comment by scottleibrand** [comment]
LOL, nice.  Thanks much, merged.

**Pull request merged**